ui/apps/platform/src/Containers/Dashboard/Widgets/ComplianceLevelsByStandard.tsx

```diff
@@ -204,10 +194,35 @@ function ComplianceLevelsByStandard() {
                 </Flex>
             }
         >
-            {complianceData && complianceData.length > 0 ? (
-                <ComplianceLevelsByStandardChart complianceData={complianceData} />
+            {Array.isArray(complianceData?.complianceLevelsByStandard) &&
+            complianceData.complianceLevelsByStandard.length !== 0 ? (
```

Comment:
just curious, why do these conditionals vs the ones it was before?

Reply:
Good question. There are 2 changes:
* Original `complianceData` was an array, now it is an object with 2 array properties.
* Although truthy condition `complianceData` would be enough for TypeScript to infer that if the object exists, then the array property exists, it seemed more explicit condition might be clearer for readers.